src/core/MOM.F90

```diff
@@ -2933,7 +2967,7 @@ subroutine extract_surface_state(CS, sfc_state)
     !$OMP parallel do default(shared)
     do j=js,je ; do i=is,ie
       ! Convert from gSalt to kgSalt
-      sfc_state%salt_deficit(i,j) = 1000.0 * US%R_to_kg_m3*US%Z_to_m*CS%tv%salt_deficit(i,j)
+      sfc_state%salt_deficit(i,j) = 0.001 * US%R_to_kg_m3*US%Z_to_m*CS%tv%salt_deficit(i,j)
```

Comment:
This is a bugfix, right?  Is the old expression not needed here?

Reply:
@Hallberg-NOAA has commented that he has no knowledge of anyone using this, and that the error is so egregious that it's almost certainly unused by anyone, so no need to retain the old behavior.